spec/requests/preferences/preference_spec.rb

```diff
@@ -81,4 +81,60 @@
       end
     end
   end
+
+  describe 'DELETE destroy' do
+    subject { delete preference_path(preference) }
+
+    let!(:user) { create(:user) }
+    let!(:preference) { create(:preference, user_id: user.id) }
+
+    context 'when logged in' do
+      before do
+        sign_in user
+      end
+
+      it 'destroys the preference' do
+        expect {
+          delete preference_path(preference)
```

Comment:
We could use `subject` here

Reply:
check that there other places we could do same change

Reply:
That's right! I'll implement it in a new commit.